dijkstra_bidirectional.go

```diff
@@ -101,12 +98,22 @@ func (graph *Graph) directionalSearch(d direction, q *vertexDistHeap, localProce
 		} else {
 			vertexList = graph.Vertices[vertex.id].inIncidentEdges
 		}
+
+		distance, ok := localQueryDist[vertex.id]
```

Comment:
I'm wondering actually, if this lookup is necessary at all, it should be the same as `vertex.dist` if I'm not mistaken (although that would be a more dangerous change).